src/WebJobs.Script/Diagnostics/StructuredLogWriter.cs

```diff
@@ -22,7 +22,7 @@ public sealed class StructuredLogWriter : IDisposable
         public StructuredLogWriter(IScriptEventManager eventManager, string baseLogPath)
         {
             string logPath = Path.Combine(baseLogPath, "structured");
-            _traceWriter = new FileTraceWriter(logPath, TraceLevel.Verbose, s => s);
+            _traceWriter = new FileTraceWriter(logPath, TraceLevel.Verbose, LogType.Structured, s => s);
```

Comment:
Not really sure what this structured log writer is doing with the identity formatter `s => s`

Reply:
This avoids the prefix that was added by the default implementation of the `FileTraceWriter`. Looking at your changes, I just noticed that this was broken in f46e487b (I guess we have a test gap), as the level was added to the wrong place. 

Any additional things we add to the message should be done in the `FormatMessage` method, so both, the the level and the newly added prefix would need to be moved there, otherwise we lose the ability to strip that information with the formatter when that is not desired.

Reply:
I fixed the previous break, ensuring that all the formatting happens in one place